README

```diff
@@ -34,6 +34,56 @@ xtensa-byt-elf-objcopy -I binary -O elf32-xtensa-le -B xtensa image.bin image.bi
 
 xtensa-byt-elf-objdump -D image.bin.elf > image.dis.txt
 
+
+logger
+======
+
+logger is used to print logs delivered from FW dma_trace mechanism, by searching log
+entries in *.ldc file generated by rimage
```

Comment:
Since you're renaming logger binary name from logger to sof_logger in PR #105, please make sure the binary name in this document has an update as well.

Reply:
Yes, I remember about it. I will update binary instruction name after PR #105 approval.